SU2_CFD/src/solvers/CFEASolver.cpp

```diff
@@ -861,22 +846,14 @@ void CFEASolver::Compute_StiffMatrix(CGeometry *geometry, CNumerics **numerics,
     LinSysRes.SetValZero();
     Jacobian.SetValZero();
 
-#ifdef HAVE_OMP
-    /*--- Chunk size is at least OMP_MIN_SIZE and a multiple of the color group size. ---*/
-    auto chunkSize = roundUpDiv(OMP_MIN_SIZE, ColorGroupSize)*ColorGroupSize;
+    for(auto color : ElemColoring) {
 
-    /*--- Loop over element colors. ---*/
-    for (auto color : ElemColoring)
-    {
-      SU2_OMP_FOR_DYN(chunkSize)
+      /*--- Chunk size is at least OMP_MIN_SIZE and a multiple of the color group size. ---*/
+      SU2_OMP_FOR_DYN(roundUpDiv(OMP_MIN_SIZE, ColorGroupSize)*ColorGroupSize)
       for(auto k = 0ul; k < color.size; ++k) {
 
         auto iElem = color.indices[k];
-#else
-    /*--- Natural coloring. ---*/
-    {
-      for (auto iElem = 0ul; iElem < nElement; iElem++) {
-#endif
+
```

Comment:
I've done some cleanup, to avoid this #ifdef regions in loops with grid coloring, for readability and so on.